atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SerializableTransaction.java

```diff
@@ -660,6 +669,12 @@ private void verifyColumnRanges(Transaction readOnlyTransaction) {
         }
     }
 
+    private Meter getTransactionConflictsMeter() {
+        // TODO(hsaraogi): add table names as a tag
+        return metricRegistry.meter(
+                MetricRegistry.name(SerializableTransaction.class, "SerializableTransactionConflict"));
+    }
```

Comment:
(There isn't really a good place to put this - do we need to mark this meter in the anonymous class's `getCommitTimestamps()` (L771-773)?)

Also see later comment on refactoring `TSCE.create()` and `mark()` to the same method

Reply:
Done.